sunpy/map/mapbase.py

```diff
@@ -193,7 +193,32 @@ def __init_subclass__(cls, **kwargs):
         if cls.__doc__ is None:
             # Set an empty string, to prevent an error adding None to str in the next line
             cls.__doc__ = ''
-        cls.__doc__ += textwrap.indent(_notes_doc, "    ")
+
+        def notes_fix(notes_doc, datadoc):
```

Comment:
Notes_doc is for the original class and datadoc is for the new subclass?

Reply:
yes!